docs/site/HasOne-relation.md

```diff
@@ -528,6 +591,131 @@ Here is a diagram to make this more intuitive:
 ### Query multiple relations
```

Comment:
@agnes512 should I remove everything below this? This is already there under hasMany.

Reply:
Let's keep them cause users might check the `HasOne` page without reading `HasMany`.

Reply:
Should I add a link to the hasMany file?

Reply:
It's fine. Three relations all have the section.

Reply:
Oh ok then.